### ISSUE TITLE
Bug: Cannot set inactive colors

### ISSUE THREAD

**Issue by zeitchef** [hoob3rt/lualine.nvim#326]
I'm trying to override the inactive colors. Currently this is my config:

```
custom_theme.inactive.c.fg = theme.blue
custom_theme.inactive.c.bg = theme.bg
```

This occasionally does work, but mostly I get the following error:

`E5113: Attempt to index field 'inactive' (a nil value)`

How can I appropriately override these values?

**Comment by shadmansaleh** [comment]
What theme are you using ?

**Comment by zeitchef** [comment]
nord

**Comment by shadmansaleh** [comment]
Are you using  nord theme that is provided with lualine or some other version for a colorscheme or somethig?

The one in this repo defines a,b,c for inactuve that overwrite should work .

https://github.com/hoob3rt/lualine.nvim/blob/9726824f1dcc8907632bc7c32f9882f26340f815/lua/lualine/themes/nord.lua#L23-L27

Check if your colorscheme or some other plugin you have is providing nord theme . Also test with [shadmansaleh/lualine.nvim](https://github.com/shadmansaleh/lualine.nvim)

**Comment by zeitchef** [comment]
Yes, I'm using the lualine-provided nord theme to override other colors as well, which work just fine.

Here's a more complete version of my setup (still shortened for brevity):

```
local nord = require 'themes.nord'
local custom_nord = require 'lualine.themes.nord'

custom_nord.normal.a.fg = nord.bg
custom_nord.normal.a.bg = nord.green
...
custom_nord.inactive.c.fg = nord.blue
custom_nord.inactive.c.bg = nord.bg

require('lualine').setup {
  options = {
    theme = custom_nord,
    ...
}
```
What am I missing here?

**Comment by zeitchef** [comment]
@shadmansaleh I also have the issue when using `shadmansaleh/lualine.nvim`

**Comment by shadmansaleh** [comment]
What's the themes.nord in here ?

```lua
local nord = require 'themes.nord'
```

**Comment by zeitchef** [comment]
This is my own module containing `nord` colors to be used globally. I don't believe this is the issue.

**Comment by shadmansaleh** [comment]
> This is my own module containing nord colors to be used globally.

Can you share it so I can test out your config ?

**Comment by zeitchef** [comment]
Here you go...

```
-- https://www.nordtheme.com/

local colors = {
  [00] = '#2e3440',
  [01] = '#3b4252',
  [02] = '#434c5e',
  [03] = '#4c566a',
  [04] = '#d8dee9',
  [05] = '#e5e9f0',
  [06] = '#eceff4',
  [07] = '#8fbcbb',
  [08] = '#88c0d0',
  [09] = '#81a1c1',
  [10] = '#5e81ac',
  [11] = '#b5616a',
  [12] = '#d08770',
  [13] = '#ebcb8b',
  [14] = '#a3be8c',
  [15] = '#b48ead',

  bg = '#2e3440',
  fg = '#d8dee9',

  black = '#2e3440',
  gray1 = '#3b4252',
  gray2 = '#434c5e',
  gray3 = '#4c566a',
  white1 = '#d8dee9',
  white2 = '#e5e9f0',
  white3 = '#eceff4',
  teal = '#8fbcbb',
  aquamarine = '#88c0d0',
  blue = '#81a1c1',
  dark_blue = '#5e81ac',
  red = '#b5616a',
  orange = '#d08770',
  yellow = '#ebcb8b',
  green = '#a3be8c',
  purple = '#b48ead',
}

return colors
```

**Comment by shadmansaleh** [comment]
Your config is working correctly without any errors . Do you lazy load lualine ? Is it garanteed lualine is in rtp when this part of your config is ran ? 

Can you put 
```lua
print(vim.inspect(custom_nord))
```
After 

> local custom_nord = require 'lualine.themes.nord'


And see if inactive is available there ?

**Comment by zeitchef** [comment]
I do lazy-load it but it is for sure in rtp. On inspection, the inactive values were available and updated. Everything is currently working as I expected.

Weird. I'll keep an eye on this and report if the problem occurs again.

**Comment by shadmansaleh** [comment]
> I do lazy-load it 

Lazy loading statusline plugin doesn't make much sense as you need it at startup .

Also if you use [shadmansaleh/lualine.nvim](https://github.com/shadmansaleh/lualine.nvim) it loads under 1ms in any decent machine . You're only asking for trouble by lazy loading it for such little gain as lots of strange bugs can pop up if it's not loaded correctly.

Also if you want faster startup take a look at [impatient.nvim](https://github.com/lewis6991/impatient.nvim) .
